lib/game.rb

```diff
@@ -0,0 +1,14 @@
+
+require_relative 'player.rb'
+require_relative 'board.rb'
+
+class Game
```

Comment:
This class should hold the main game logic. The game loop, game start should be written here. This is what OOP is about, you have the OOP arrangement but not the functionality